### PR TITLE
Add a CLI command for retrieving the autopilot configuration.

### PR DESCRIPTION
**Pull request by mkeeler**
This is a small shim around the new `/v1/operator/autopilot/state` endpoint. The new command supports both JSON and pretty output formats.

### CODE REVIEW

File: website/pages/commands/operator/autopilot.mdx [hunk 1]
```diff
@@ -97,3 +97,72 @@ Configuration updated!
...
+
+@include 'http_api_options_server.mdx'
+
+#### Command Options
+
+- `-format` - Specifies the output format. Must be one of `[pretty|json]` and it defaults to `pretty`.
+
+#### Sample Output
+
+```
```
Comment (s-christoff) [inline website/pages/commands/operator/autopilot.mdx:119]: Nit: I think it's really nice to show the command ran in the sample output if possible

**Review by s-christoff — APPROVED** [review]
Everything looks great, I hardly have any comments. Aside from the thing we discussed in Slack I think my only concern is that locally I am getting remnants of enterprise features in my output, however this is probably just be a *me* issue. Just wanted to bring it up in case it shows up. 
I'm going to approve this now since there is only 1 small thing needed to be added and then it should be good.

**Comment by mkeeler** [comment]
The output contains 2 things that would look enterprise related. the first is the optimistic failure tolerance and the second is the network segment showing up in the server Meta.

The empty network segment meta is due to Consul OSS always populating that here:

https://github.com/hashicorp/consul/blob/3760e3d12db55e9497a7fd3c8ef2e26c948439cc/agent/agent.go#L3402

The optimistic failure tolerance which really only applies when redundancy zones are in use is due to the field in the `api.AutopilotState` struct being an `int` and that 0 is a valid value when the system cannot tolerate more failures. So we cannot just tag the field with a `omitempty` JSON tag. We could turn this field into a pointer in which case we could detect being unset vs a zero value but the ergonomics of using the struct would be diminished.

File: command/operator/autopilot/state/formatter.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+		buffer.WriteString(fmt.Sprintf("%s%s\n", indent, val))
+	}
+}
+
+func (f *prettyFormatter) FormatState(state *api.AutopilotState) (string, error) {
+	var buffer bytes.Buffer
+
+	buffer.WriteString(fmt.Sprintf("Healthy:                      %t\n", state.Healthy))
+	buffer.WriteString(fmt.Sprintf("Failure Tolerance:            %d\n", state.FailureTolerance))
+	buffer.WriteString(fmt.Sprintf("Optimistic Failure Tolerance: %d\n", state.OptimisitcFailureTolerance))
```
Comment (crhino) [inline command/operator/autopilot/state/formatter.go:58]: I realized this when looking at the output, we've misspelled `Optimistic` in the api models as `Optimisitc`!

Comment (mkeeler) [reply command/operator/autopilot/state/formatter.go]: Yep, I saw that too a few minutes ago and am fixing it.

**Pull request merged**